docs/how-to-use-plugins-with-ergoCub.md

```diff
@@ -0,0 +1,190 @@
+# How to use `gz‐sim‐yarp‐plugins` with ergoCub
+
+This document explains how to install and use ergoCub in the Modern Gazebo (gz-sim) simulator through `gz-sim-yarp-plugins`.
+
+> [!NOTE]
+> For the time being this guide will provide operative instructions only for **Linux** distributions; instructions will also be provided for Windows and macOS in the future.
+
+Note that some features of the ergocub model (mainly the depth sensors and the hands) are not supported, see <https://github.com/robotology/gz-sim-yarp-plugins/issues/137> for more details.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [World creation](#world-creation)
+- [Run the simulation](#run-the-simulation)
+
+## Installation
+
+The easiest ways to setup an environment with everything we need are:
+
+- Using the robotology-superbuild
+- Creating a conda environment
+
+### Conda environment
+
+#### Install a conda distribution
+
+If you do not have one, please install a conda distribution. We suggest to use the minimal [mambaforge](https://github.com/conda-forge/miniforge#mambaforge) distribution, which uses conda-forge packages by default and installs the [mamba](https://github.com/mamba-org/mamba) command by default.
+
+To install mambaforge, please follow the instructions in [install-mambaforge](https://github.com/robotology/robotology-superbuild/blob/master/doc/install-mambaforge.md) documentation.
```

Comment:
```suggestion
If you do not have one, please install a conda distribution. We suggest to use the minimal [miniforge](https://github.com/conda-forge/miniforge) distribution, which uses conda-forge packages by default.

To install miniforge, please follow the instructions in [install-miniforge](https://github.com/robotology/robotology-superbuild/blob/master/doc/install-miniforge.md) documentation.
```

mambaforge is deprecated and is going to be removed soon: https://github.com/conda-forge/miniforge?tab=readme-ov-file#download, let's try to document the use of miniforge.